tools/bazel_rules/jest/.eslintrc.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
We're not using eslint in barista. This is probably an artifact from copying it, right?
Can you make sure that linting is set up correctly for this project. 

With this, you should see the license header failing, as this is not yet changed in barista. 
I'm opening the PR right now.